tools.json

```
@@ -127,6 +127,14 @@
     "description": "Ruby/Redis based recommendation engine (collaborative filtering).",
     "authors": ["paulasmuth"]
   },
+  {
+    "name": "redli",
+    "language": "Go",
+    "url": "https://github.com/IBM-Cloud/redli",
+    "repository": "https://github.com/IBM-Cloud/redli",
```

I don't think there is a benefit adding the same URL twice. People would click on both just to find out they point to the same page.

```suggestion
    "repository": "https://github.com/IBM-Cloud/redli",
```